tools/src/launcher/Main.hs

```diff
@@ -13,64 +13,70 @@
-import           Pos.DB.Block (dbGetSerBlockRealDefault, dbGetSerUndoRealDefault,
-                               dbPutSerBlundRealDefault)
-import           Pos.DB.Class (MonadDB (..), MonadDBRead (..))
-import           Pos.DB.Rocks (NodeDBs, closeNodeDBs, dbDeleteDefault, dbGetDefault,
-                               dbIterSourceDefault, dbPutDefault, dbWriteBatchDefault, openNodeDBs)
-import           Pos.Launcher (HasConfigurations, withConfigurations)
-import           Pos.Launcher.Configuration (ConfigurationOptions (..))
-import           Pos.Reporting.Methods (compressLogs, retrieveLogFiles, sendReport)
-import           Pos.ReportServer.Report (ReportType (..))
-import           Pos.Update (installerHash)
-import           Pos.Update.DB.Misc (affirmUpdateInstalled)
-import           Pos.Util (HasLens (..), directory, postfixLFields, sleep)
-import           Pos.Util.CompileInfo (HasCompileInfo, retrieveCompileTimeInfo, withCompileInfo)
+import           Control.Exception                    (handle, mask_, throwIO)
+import           Foreign.C.Error                      (Errno (..), ePIPE)
+import           GHC.IO.Exception                     (IOErrorType (..), IOException (..))
+
+import           Paths_cardano_sl                     (version)
+import           Pos.Client.CLI                       (configurationOptionsParser, readLoggerConfig)
+import           Pos.Core                             (HasConfiguration, Timestamp (..))
+import           Pos.DB.Block                         (dbGetSerBlockRealDefault,
+                                                       dbGetSerUndoRealDefault,
+                                                       dbPutSerBlundRealDefault)
+import           Pos.DB.Class                         (MonadDB (..), MonadDBRead (..))
+import           Pos.DB.Rocks                         (NodeDBs, closeNodeDBs, dbDeleteDefault,
+                                                       dbGetDefault, dbIterSourceDefault,
+                                                       dbPutDefault, dbWriteBatchDefault,
+                                                       openNodeDBs)
+import           Pos.Launcher                         (HasConfigurations, withConfigurations)
+import           Pos.Launcher.Configuration           (ConfigurationOptions (..))
+import           Pos.Reporting.Methods                (compressLogs, retrieveLogFiles, sendReport)
+import           Pos.ReportServer.Report              (ReportType (..))
+import           Pos.Update                           (installerHash)
+import           Pos.Update.DB.Misc                   (affirmUpdateInstalled)
+import           Pos.Util                             (HasLens (..), directory, postfixLFields,
+                                                       sleep)
+import           Pos.Util.CompileInfo                 (HasCompileInfo, retrieveCompileTimeInfo,
+                                                       withCompileInfo)
```

Comment:
Setup stylish-haskell properly please. I can't accept these changes in import section. See also: https://github.com/input-output-hk/cardano-sl/pull/1629#discussion_r155807686